cloud/amazon/ec2_vpc_igw.py

```diff
@@ -20,18 +20,25 @@
 description:
     - Manage an AWS VPC Internet gateway
 version_added: "2.0"
-author: Robert Estelle, @erydo
+author: Robert Estelle (@erydo)
 options:
   vpc_id:
     description:
       - The VPC ID for the VPC in which to manage the Internet Gateway.
     required: true
     default: null
+  region:
+    description:
+      - The AWS region to use.  Must be specified if ec2_url is not used. If not specified then the value of the EC2_REGION environment variable, if any, is used. See U(http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region)
```

Comment:
" Must be specified if ec2_url is not used." should probably be removed - as you rightly point out, it could be in boto profile - I realise this is likely wrong in a lot of other modules too.